Seeq.Link.SDK.Debugging.Agent/EntryPoint.cs

```diff
@@ -12,11 +16,31 @@ namespace Seeq.Link.Debugging.Agent {
     /// to connect to the server and load the connector that is under development.
     /// </summary>
     public class EntryPoint {
+        private const string AGENT_ONE_TIME_PASSWORD_PLACEHOLDER = "<your_one_time_password>";
 
         public static void Main(string[] args) {
-            log4net.Config.XmlConfigurator.Configure();
+            XmlConfigurator.Configure();
 
-            Seeq.Link.Agent.Program.Configuration config = Seeq.Link.Agent.Program.GetDefaultConfiguration();
+            const string agentName = ".NET Connector SDK Debugging Agent";
+            var executingAssemblyLocation = Assembly.GetExecutingAssembly().Location;
+            var seeqDataFolder = Path.Combine(Path.GetDirectoryName(executingAssemblyLocation), "data");
+
+            // change this value to the one-time password generated from the Agents tab of your Seeq server's Administration
+            // page providing the machine name and your connector's name
+            const string agentOneTimePassword = AGENT_ONE_TIME_PASSWORD_PLACEHOLDER;
+
+            if (agentOneTimePassword != AGENT_ONE_TIME_PASSWORD_PLACEHOLDER) {
+                var agentHelper = new AgentHelper(agentName);
+                var secretsPath = Path.Combine(seeqDataFolder, SeeqNames.Agents.AgentKeysFolderName, "agent.keys");
+                var secretsManager = new FileBasedSecretsManager(secretsPath);
+
+                // set the agent's pre-provisioned one-time password
+                var preProvisionedOneTimePasswordSecretName =
+                    $"{agentHelper.ProvisionedAgentUsername}|PRE_PROVISIONED_ONE_TIME_PASSWORD";
+                secretsManager.PutSecret(preProvisionedOneTimePasswordSecretName, agentOneTimePassword);
```

Comment:
Do we want to put the secret one time password everytime  in the vault? Do we need to do that? It looks a bit strange to me.
I think it would be nearer to the normal workflow if we just put the secret in case it is not already there.

Maybe it would make sense to have the secret in a file and once it is used to just be emptied?

Alternativelly: to have another "main" program that will set the OTP and this EntryPoint will just assume the secret is already configured.